households.py

```diff
@@ -309,16 +309,17 @@ def create_output_zip(args, n_households, household_time):
 
     metadata["number_of_households"] = n_households
 
+    metadata["household_garble_time"] = household_time.isoformat()
+
     if not args.householddef:
-        metadata["household_inference_time"] = household_time.isoformat()
         metadata["households_inferred"] = True
     else:
         metadata["households_inferred"] = False
 
-    with open(Path("temp-data") / new_metadata_filename, "w") as metadata_file:
+    with open(Path("temp-data") / new_metadata_filename, "w+") as metadata_file:
```

Comment:
Can you explain the reason for this change (`w` -> `w+`)? `+` just enables reading the file, which this code doesn't seem to do.

Also, I know this code was introduced in a separate PR, but is there a reason we save this metadata file in both `temp-data/` and `output/`?

Reply:
Not sure on why w -> w+ is needed, but I think the reason for the the metadata being in both directories is the `temp-data` folder is meant to be more persistent between steps (despite the name). So the output folder isn't expected to last after things are zipped, but the temp-data folder will contain the original pii file and metadata for later steps to refer back to